app/models/concerns/non_null_session_uuid.rb

```diff
@@ -1,11 +0,0 @@
-module NonNullSessionUuid
-  extend ActiveSupport::Concern
-
-  included do
-    before_create :generate_session_uuid
-  end
-
-  def generate_session_uuid
-    self.session_uuid = '_' + SecureRandom.uuid
-  end
-end
```

Comment:
Also a separate PR for this maybe? (sorry to burden you but I really like small PRs).